program/ConvertMinutesToHours.js

```diff
@@ -0,0 +1,13 @@
+// 60 Minutes = 1 Hour
+
+function ConvertMinutesToHours(minute){
+    let hours = minute/60
+    return hours;
+}
+
+// Test Console
+console.log(`${ConvertMinutesToHours(300)}Hours`);
+
+console.log(`${ConvertMinutesToHours(120)}Hours`);
+
+console.log(`${ConvertMinutesToHours(60)}Hours`);
```

Comment:
The output is `1hours`.  You can make it output `hour` or `hours` based on the result you get.